Doc/howto/mro.rst

```diff
@@ -0,0 +1,758 @@
+.. _python_2.3_mro:
+
+The Python 2.3 Method Resolution Order
```

Comment:
I understand that for this PR you might want to simply move the document unchanged, but if this is the current default, maybe the title should be updated.  This title makes me thing it's an historical document that describes the MRO as it was in Python 2.3.

The markup, formatting, and tone also need some tweaking, or a note about the origin of this document should be added at the top.

Reply:
Yes, this is a historical document. I don't want to rewrite its content or tone, but keep it as an article that was written at that time. Markup and formatting can be adjusted.